packages/components/src/alignment-matrix-control/index.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+import classnames from 'classnames';
+import {
+	unstable_useCompositeState as useCompositeState,
+	unstable_Composite as Composite,
+	unstable_CompositeGroup as CompositeGroup,
+} from 'reakit';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useInstanceId } from '@wordpress/compose';
+import { useState, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Cell from './cell';
+import { Root, Row } from './styles/alignment-matrix-control-styles';
+import { useRTL } from '../utils/rtl';
+import AlignmentMatrixControlIcon from './icon';
+import { GRID, getItemId } from './utils';
+
+function useBaseId( id ) {
+	const instanceId = useInstanceId( AlignmentMatrixControl );
+	const prefix = id || 'alignment-matrix-control';
+
+	return `${ prefix }-${ instanceId }`;
+}
+
+export default function AlignmentMatrixControl( {
+	className,
+	id,
+	label = __( 'Alignment Matrix Control' ),
+	hasFocusBorder = true,
+	defaultValue = 'center center',
+	value,
+	onBlur = noop,
+	onChange = noop,
+	width = 92,
+	...props
+} ) {
+	const [ immutableDefaultValue ] = useState( value ?? defaultValue );
+	const isRTL = useRTL();
+	const baseId = useBaseId( id );
+	const initialCurrentId = getItemId( baseId, immutableDefaultValue );
+
+	const composite = useCompositeState( {
+		baseId,
+		currentId: initialCurrentId,
+		rtl: isRTL,
+		unstable_virtual: true,
```

Comment:
If items need to show tooltips, I would recommend not to use `aria-activedescendant` here (by removing this `unstable_virtual: true`). It would also make the `handleOnBlur` workaround unnecessary.

Reply:
That works @diegohaz !! Thank you 🙏